app/screens/image_preview/image_preview.js

```diff
@@ -375,11 +375,12 @@ export default class ImagePreview extends PureComponent {
             const flattenStyle = StyleSheet.flatten(style);
             const calculatedDimensions = calculateDimensions(height, width, deviceWidth, deviceHeight - statusBar);
             const imageStyle = {...flattenStyle, ...calculatedDimensions};
+            const src = {...source, cache: FastImage.cacheControl.cacheOnly};
```

Comment:
Why not get from original source if (somehow) cached version doesn't exist?

Reply:
if cache version does not exist at this point we have an error in logic elsewhere.. this is for the gallery view and I do expect to have it cached by this point